csharp/ql/test/library-tests/csharp8/NullableRefTypes.ql

```diff
@@ -0,0 +1,17 @@
+import csharp
+
+query predicate suppressNullableWarnings(SuppressNullableWarningExpr e, Expr op) {
```

Comment:
Need to add an exclusion in the nullness library that this is a non-null expression.